### PR TITLE
[Test/NNFW] separate testcase

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. separate testcase - invalid model and invalid data.
2. fix mem leak, destroy handle before creating new data.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc [hunk 1]
```diff
@@ -748,7 +748,7 @@ TEST (nnstreamer_nnfw_mlapi, invoke_pipeline_00)
   size_t data_size;
 
   gchar *test_model;
-  guint *sink_called_cnt  = (guint *) g_malloc (sizeof (guint));
+  guint *sink_called_cnt = (guint *) g_malloc0 (sizeof (guint));
```
Comment (dongju-chae) [inline tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc:751]: Is there any reason to use a pointer of `guint`? What happens if just using `guint sink_called_cnt = 0;`

Comment (jaeyun-jung) [reply tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc:751]: to avoid reference of local variable in callback.

Comment (wooksong) [inline tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc:751]: The type casting might be unnecessary here.

> A pointer to void may be converted to or from a pointer to any object type. A pointer to any object type may be converted to a pointer to void and back again; the result shall compare equal to the original pointer.

Comment (jaeyun-jung) [reply tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc:751]: But, this makes build failure:
```
error: invalid conversion from 'gpointer' {aka 'void*'} to 'guint*' {aka 'unsigned int*'} [-fpermissive]
```

**Pull request merged**